### PR TITLE
Add Play User Group Austria

### PR DESCRIPTION
**Pull request by mkurz**
@domdorn founded a user group in Austria.
I put it beneath Berlin and Cologne because Vienna is german speaking as well :wink:

### CODE REVIEW

File: documentation/manual/about/PlayUserGroups.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ http://www.meetup.com/Play-Berlin-Brandenburg/
 
 [Xing](https://www.xing.com/communities/groups/scala-user-group-koeln-bonn-1035441) / [Twitter](https://twitter.com/scalacgn)
 
+## Vienna - AUSTRIA
+
+http://www.playframework.wien
```
Comment (marcospereira) [inline documentation/manual/about/PlayUserGroups.md]: This is redirecting to https://www.meetup.com/PlayFramework-Wien/. Should use this final url.

Comment (mkurz) [inline documentation/manual/about/PlayUserGroups.md]: @domdorn What do you think?

**Comment by domdorn** [comment]
please use 
http://playframework.wien instead of the Meetup url.
I was thinking of setting up a website with additional content. The redirect is just in the meantime.

**Comment by mkurz** [comment]
I removed the `www` from the url.

**Pull request merged**